### PR TITLE
[CI] Fix the compilation of relation_algebra.

### PR DESCRIPTION
**Pull request by ppedrot**
For some reason it is now requiring the AAC plugin.

### CODE REVIEW

**Comment by Alizter** [comment]
@ppedrot That didn't seem to work, although I am unsure why.

**Comment by ppedrot** [comment]
Ping @coq/ci-maintainers they might now why.

File: .gitlab-ci.yml
Comment (SkySkimmer) [inline .gitlab-ci.yml:883]:
```suggestion
  - build:base
  - plugin:ci-aac_tactics
```

Comment (ppedrot) [reply .gitlab-ci.yml:883]: Duh.

**Comment by ppedrot** [comment]
And now we have
```
Error: Cannot find a physical path bound to logical path all_ssreflect with prefix mathcomp.
```
This development is making big changes in our back!

**Comment by ppedrot** [comment]
And now
```
The file /builds/coq/coq/_build_ci/mathcomp/mathcomp/ssreflect/all_ssreflect.vo was compiled with OCaml 4.14.0 while this instance of Coq was compiled with OCaml 4.09.0.
```
Gah, why is there so much hidden state in the CI script?

**Comment by Zimmi48** [comment]
> And now
> ```
> The file /builds/coq/coq/_build_ci/mathcomp/mathcomp/ssreflect/all_ssreflect.vo was compiled with OCaml 4.14.0 while this instance of Coq was compiled with OCaml 4.09.0.
> ```
> Gah, why is there so much hidden state in the CI script?

This probably means that part of the dependencies use the edge flambda compiler and part use the base compiler. Make sure everyone in the connex component of the dependency happy uses the edge compiler.

**Comment by Zimmi48** [comment]
> And now we have
> ```
> Error: Cannot find a physical path bound to logical path all_ssreflect with prefix mathcomp.
> ```
> This development is making big changes in our back!
> 

@damien-pous any idea what's going on with the dependencies?

**Comment by ppedrot** [comment]
CI job for relation algebra green, at least it seems to have worked!

**Comment by SkySkimmer** [comment]
Looks like https://github.com/damien-pous/relation-algebra/commit/a41bc9ae154c3ddcde17bb3ed739ef02ccc6e8e4 changed the default files from default off to default on.

**Review by Zimmi48 — APPROVED** [review]
Let's merge to fix the CI and we can always revert later if @damien-pous returns to the previous list of files to build by default.

**Comment by Zimmi48** [comment]
@coqbot merge now

**Comment by damien-pous** [comment]
Hi, indeed I have changed the optional files from default off to default on, in the 1.7.8 release.
This was not intentional, though. I you want me to revert to the old behaviour I can. 
(But I wouldn't do a release just for that I guess).

**Comment by Zimmi48** [comment]
The Coq CI is fixed, so it's no longer a problem for us. It's your call whether you change it back, but if you did, I would recommend making a new release ASAP. It would be very strange for your users (through opam) to have a back and forth between releases that are evenly spaced in time. In particular, if you decided to revert to the old behavior, you should not leave the 1.7.8 version be the one that will be in the next Coq Platform release. (cc @MSoegtropIMC)

**Comment by SkySkimmer** [comment]
The relation-algebra opam package calls configure so probably isn't affected by the default changes.

**Comment by SkySkimmer** [comment]
Indeed observe https://gitlab.com/coq/opam-coq-archive/-/jobs/2998974978 where it installs without the optional deps present.

**Comment by Zimmi48** [comment]
Oh, OK, then another fix for us would have been to call configure? In any case, I retract what I wrote above then.

**Comment by damien-pous** [comment]
Calling the configure script with --enable/disable-ssr/aac let's you decide explicitly, indeed.
I feel the current option is better, anyways: it's good that CI checks the optional files, via requiring the optional deps.

**Pull request merged**
